xbmc/ColorManager.cpp

```diff
@@ -0,0 +1,531 @@
+#include <boost/algorithm/clamp.hpp>
```

Comment:
Looks like VideoRederer is the only client of this class. Then it should be moved to this component.